src/main/java/lab5/reflections.md

```diff
@@ -0,0 +1,99 @@
+Ques.1   Why a HumanPlayer can be returned when the return type is Player. ?
+Ans....  This is possible because of polymorphism in Java, specifically subtype polymorphism (also known as dynamic method dispatch).
```

Comment:
subtype polymorphism and dynamic method dispatch are related but different concepts.  You are discussing subtype polymorphism here.